.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -34,7 +34,7 @@ etc.
 **Linux or Windows?**
 <Linux/Windows>
 
-**EC2Fleet Configuration as Code**
```

Comment:
Lets keep this `EC2Fleet` because the the CasC config uses `ec2Fleet` and we are not changing that.

Reply:
Reverted name change

Reply:
Don't think this change was reverted. Unresolving conv to make sure the change gets done.